### DISCUSSION TITLE
Headless2 not running in headless

### DISCUSSION THREAD

**Discussion by randonbr** [seleniumbase/SeleniumBase#3066] [Q&A] [answered]
I am trying to run a headless mode execution using headless2 on SB.

While the code executes, the execution is not headless. The chrome browser window pops up as in a non-headless mode execution.

Code:
`
from seleniumbase import SB

class TestHeadlessMode:
    def test_example(self, sb: SB):
        sb.open("https://www.example.com")
        sb.assert_title("Example Domain")
if __name__ == "__main__":
    sb = SB()
    sb.run_tests([TestHeadlessMode], headless2=True)
`

**Comment by mdmintz** [comment, marked as answer]
To specify `headless`, use `headless=True` in your `SB()` call. Eg. `SB(headless=True)`.

See [The 23 Syntax Formats](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md) for structuring your `SeleniumBase` scripts properly.
